site/runSiteNavigation.tsx

```diff
@@ -0,0 +1,23 @@
+import ReactDOM from "react-dom"
+import { getOwidGdocFromJSON, OwidGdocType } from "@ourworldindata/utils"
+import { SiteNavigation } from "./SiteNavigation"
+
+export const runSiteNavigation = (
+    baseUrl: string,
+    hideDonationFlag?: boolean
+) => {
+    // Used to determine whether or not to show the searchbar in the header
+    let isOnHomepage = false
+    if (window._OWID_GDOC_PROPS) {
+        const props = getOwidGdocFromJSON(window._OWID_GDOC_PROPS)
+        isOnHomepage = props?.content?.type === OwidGdocType.Homepage
+    }
+    ReactDOM.render(
+        <SiteNavigation
```

Comment:
Technically we don't need to fetch the `topicTagGraph.json` when we're on the homepage, because it's already embedded in the homepage gdoc's `homepageMetadata`, but seeing as the JSON request is ~4kB transferred, I think it's okay to not optimize.

Later, we should see if it's possible to get this file and `dods.json` to cache.